client/ayon_core/pipeline/workfile/utils.py

```diff
@@ -417,7 +417,9 @@ def save_next_version(
     Args:
         version (Optional[int]): Workfile version that will be used. Last
             version + 1 is used if is not passed in.
-        comment (optional[str]): Workfile comment.
+        comment (optional[str]): Workfile comment. Pass '""' to clear comment.
+            The last workfile comment is used if it is not passed in and
+            passed 'version' is 'None'.
```

Comment:
I wonder if that's really the most sensible behavior - even if I were to pass a version, why wouldn't it also preserve the comment? Not a big hurdle, but sounds confusing? good to see it mentioned in the docstring though.

Reply:
Maybe doesn't make sense. My logic was "if you pass in `version`, you define the rest too", maybe that's not right...